.github/workflows/snap.yaml

```diff
@@ -0,0 +1,139 @@
+name: Build Spark client snap and run tests
+
+env:
+  VERSION: 0.1
+  RELEASE: edge
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  build:
+    name: Build Snap
+    runs-on: ubuntu-latest
+    outputs:
+      snap-file: ${{ steps.build-snap.outputs.snap }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
```

Comment:
Hi @Mehdi-Bendriss , I referred to your opensearch snap build workflow and created this draft. Please take a look.

How do I develop/test this further in a sandbox before pushing it out to github?